app/views/devise/two_step_verification_session/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for :title, "2-step verification" %>
```

Comment:
Should we get rid of `app/views/devise/two_step_verification_session/show.html.erb`?